editoast/src/views/search.rs

```diff
@@ -615,6 +615,34 @@ pub(super) struct SearchResultItemProject {
     tags: Vec<String>,
 }
 
+#[derive(Search, Serialize, ToSchema)]
+#[search(
+    table = "search_user",
+    migration(src_table = "authn_user"),
+    joins = "INNER JOIN authn_user ON authn_user.id = search_user.id",
```

Comment:
No need to join over the real table since all the necessary data to build the response item is already in cache.